packages/libs/chainweb-stream-client/src/examples/index.ts

```diff
@@ -0,0 +1,62 @@
+import ChainwebStreamClient, { ITransaction } from '../';
+
+const client: ChainwebStreamClient = new ChainwebStreamClient({
+  // required
+  type: 'account',
+  id: 'k:e7f7130f359fb1f8c87873bf858a0e9cbc3c1059f62ae715ec72e760b055e9f3',
+  host: 'http://localhost:4000/',
+
+  // optional
+  limit: 100,
+  connectTimeout: 15_000,
```

Comment:
didn't knew this was allowed :100: this makes it so much easier to read, nice!

Reply:
the _ separator in numbers? I love this feature for large numbers